src/main/java/io/quarkus/gizmo/ClassTransformer.java

```diff
@@ -0,0 +1,196 @@
+package io.quarkus.gizmo;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import org.objectweb.asm.ClassVisitor;
+import org.objectweb.asm.MethodVisitor;
+
+/**
+ * Allows transforming an existing class by wrapping an ASM {@link ClassVisitor}.
```

Comment:
`Allows transforming ...` -> `Transforms ...` or `Allows for transformation of...`?

Reply:
I spent some time thinking about this exact sentence, because I didn't like it too much either. But "Transforms an existing class" is better, I'll fix that.

Reply:
Done.